projects/coordinator/src/main/java/org/batfish/coordinator/config/Settings.java

```diff
@@ -34,9 +34,10 @@
    private static final String ARG_QUEUE_COMPLETED_WORK = "qcompletedwork";
    private static final String ARG_QUEUE_INCOMPLETE_WORK = "qincompletework";
    private static final String ARG_QUEUE_TYPE = "qtype";
-   private static final String ARG_SERVICE_HOST = "servicehost";
 
+   private static final String ARG_SERVICE_POOL_HOST = "poolhost";
```

Comment:
I was trying to keep names consistent, the surrounding names are 'servicehost', 'poolport', 'workport', I can change them to 'poolbindhost'.